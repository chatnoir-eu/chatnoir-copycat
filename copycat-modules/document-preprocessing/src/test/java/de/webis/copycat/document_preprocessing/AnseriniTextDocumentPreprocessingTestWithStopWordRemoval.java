package de.webis.copycat.document_preprocessing;

import org.approvaltests.Approvals;
import org.junit.Test;

public class AnseriniTextDocumentPreprocessingTestWithStopWordRemoval extends DocumentPreprocessingTest {

	public AnseriniTextDocumentPreprocessingTestWithStopWordRemoval() {
		super(new AnseriniDocumentTransformation("JsoupStringTransform"));
	}

	@Test
	public void approveWebisHtmlPage() {
		Approvals.verify(preprocessedHtml("webis-de"));
	}
	
	@Test
	public void approveToucheHtmlPage() {
		Approvals.verify(preprocessedHtml("touche-webis-de"));
	}
	
	@Test
	public void approveSigirHtmlPage() {
		Approvals.verify(preprocessedHtml("sigir-org-2021-resource"));		
	}
}
