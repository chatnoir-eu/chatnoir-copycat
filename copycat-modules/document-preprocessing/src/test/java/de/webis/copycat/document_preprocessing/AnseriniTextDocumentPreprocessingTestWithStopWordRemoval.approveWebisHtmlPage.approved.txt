webis people students lecturenotes research publications data events facilities webis.de people students lecturenotes research publications data events facilities information nothing without retrieval webis group addresses challenges information society conducting basic research developing technology implementing evaluating prototypes future information systems our research contributes web mining retrieval machine learning computational linguistics symbolic ai learn more search engines args argument search chatnoir web search netspeak writing assistance picapica plagiarism detection halle home people teaching research leipzig home people teaching research paderborn home people teaching research weimar home people teaching research Â© 2021 webis group contact impressum terms privacy